spec/requests/users_controller_spec.rb

```diff
@@ -1,21 +1,41 @@
 RSpec.describe UsersController, type: :request do
   describe 'GET /users' do
-    let!(:user) { create(:user, email:'miotyan@example.com') }
+    let!(:user) { create(:user) }
     let!(:another_user) { create(:user, email:'yukko@example.com') }
 
-    it 'リクエストが成功し、ユーザー名が表示されること' do
-      aggregate_failures do
-        get users_path
-        expect(response.status).to eq 200
-        expect(response.body).to include user.email
-        expect(response.body).to include another_user.email
+    context 'ログインしているかつ、正常な時' do
+      before do
+        post login_path, params: { email: user.email, password: 'sample_password' }
```

Comment:
以下のように書いた方が同じ文字列を複数回書かなくて済むからいいと思います。
```suggestion
        post login_path, params: { email: user.email, password: user.password }
```